yodeploy/application.py

```diff
@@ -48,17 +48,20 @@ def deployed_versions(self):
                        if version != 'unpack'),
                       key=version_sort_key)
 
-    def deploy_ve(self, target, repository, version):
-        '''Unpack a virtualenv for the deploy hooks, and return its location
-        on the FS
-        '''
-        deploy_req_fn = os.path.join(self.appdir, 'versions', version,
+    def deploy_ve(self, target, repository, app_version):
+        """Prepare the deploy virtualenv.
+
+        Unpack a virtualenv for the deploy hooks, and return its location on
+        the FS.
+        """
+        deploy_req_fn = os.path.join(self.appdir, 'versions', app_version,
                                      'deploy', 'requirements.txt')
-        ve_hash = yodeploy.virtualenv.sha224sum(deploy_req_fn)
-        ve_hash = yodeploy.virtualenv.ve_version(ve_hash)
+        hash_ = yodeploy.virtualenv.sha224sum(deploy_req_fn)
+        platform = self.settings.artifacts.platform
+        version = yodeploy.virtualenv.ve_version(hash_, platform)
```

Comment:
The confusion already exists since an app also has a version.

Maybe the lest ambiguous terminology would be "virtualenv identifier"